themes/hugo-rladies/layouts/directory/single.html

```diff
@@ -28,9 +28,9 @@ <h1 class="entry-title"><a href="{{ .Permalink }}" rel="bookmark">{{ .Title }}</
       
       <!-- pagination nav -->
       <div class="nav pagination justify-content-center align-content-center" aria-label="Directory pagination">
-        <a class="pagination-prev" href="#directory-content" onclick=".prev()"><</a>
+        <a class="pagination-prev disabled" href="#directory-content"><span class="page-arrow"><</span></a>
         <ul class="pagination-layout"></ul>
-        <a class="pagination-next" href="#directory-content" onclick=".next()">></a>
+        <a class="pagination-next" href="#directory-content"><span class="page-arrow">></span></a>
```

Comment:
```suggestion
        <a class="pagination-next" href="#directory-content"><span class="page-arrow"></span></a>
```

Reply:
trimming excess `>`

Reply:
This `>` is the `next` arrow button.

Reply:
hahah!of course! I was just in html tag- closing mindset!